src/frameworks/next/index.ts

```diff
@@ -1,10 +1,25 @@
 import { execSync } from "child_process";
-import { readFile, mkdir, copyFile } from "fs/promises";
+import { mkdir, copyFile } from "fs/promises";
 import { dirname, join } from "path";
 import type { NextConfig } from "next";
+import {
+  APP_PATH_ROUTES_MANIFEST,
+  EXPORT_MARKER,
+  IMAGES_MANIFEST,
+  MIDDLEWARE_MANIFEST,
+  PAGES_MANIFEST,
+  PRERENDER_MANIFEST,
+  ROUTES_MANIFEST,
+} from "next/constants";
```

Comment:
Are these going to work in prod, since next is just a dev dep? We might need to dynamically import them or bundle

Reply:
right, forgot about that. I used them as types moved all to a constants file